interface/globals.php

```diff
@@ -271,6 +271,7 @@ function undoMagicQuotes($array, $topLevel=true) {
       $gl_user[$iter]=$row;
     }
   }
+}
   // Set global parameters from the database globals table.
```

Comment:
This bracket was already included in another merged PR, wasn't it?

Reply:
I am not sure why another bracket would be needed there...
Looks like you have a branch Procedure_order, so I am not sure why you would have excess stuff unless this was a twig branch.